api/src/main/java/org/openmrs/module/appointments/service/AppointmentsService.java

```diff
@@ -38,6 +38,9 @@ public interface AppointmentsService {
     @Authorized({VIEW_APPOINTMENTS, MANAGE_APPOINTMENTS})
     List<Appointment> getAllAppointmentsReminder(String hours);
 
+    @Transactional
+    @Authorized({VIEW_APPOINTMENTS, MANAGE_APPOINTMENTS})
+    List<Appointment> getAllAppointments(Date forDate, String status);
```

Comment:
There needs to be JavaDoc _somewhere_, even better in multiple places, that describes what this method does, especially given the implementation in the DAO.

Reply:
Agree that we should add a javadoc here... it looks like there aren't any java docs on the other methods here, but let's get in the practice and add it for new methods we add or existing ones we modify.